buid	,	V_9
bus	,	V_26
rtas_pci_ops	,	V_46
out_be32	,	F_23
__iomem	,	T_2
PCIBIOS_BAD_REGISTER_NUMBER	,	V_13
pci_bus	,	V_25
dn	,	V_2
"ibm,write-pci-config"	,	L_9
dev	,	V_31
len	,	V_42
u32	,	T_1
pci_ext_config_space	,	V_4
first_busno	,	V_43
rtas_pci_read_config	,	F_8
registers	,	V_34
model	,	V_32
state	,	V_16
chip_regs	,	V_35
config_access_valid	,	F_1
val	,	V_7
ret	,	V_11
ibm_write_pci_config	,	V_27
resource	,	V_33
CONFIG_EEH	,	F_3
bus_range	,	V_41
pci_get_pdn_by_devfn	,	F_9
"reg0: %x\n"	,	L_5
"Python"	,	L_2
be32_to_cpu	,	F_29
devfn	,	V_20
rtas_call	,	F_5
ioremap	,	F_20
PCIBIOS_SET_FAILED	,	V_18
PCIBIOS_DEVICE_NOT_FOUND	,	V_12
in_be32	,	F_22
phb_set_bus_ranges	,	F_28
eeh_dev_check_failure	,	F_11
"ibm,read-pci-config"	,	L_8
ops	,	V_45
size	,	V_6
returnval	,	V_8
rtas_config_addr	,	F_4
is_python	,	F_15
ibm_read_pci_config	,	V_22
BUID_LO	,	F_7
rtas_write_config	,	F_13
get_phb_buid	,	F_27
pdn_to_eeh_dev	,	F_12
BUID_HI	,	F_6
python_countermeasures	,	F_17
"Can't get address for Python workarounds !\n"	,	L_3
edev	,	V_14
rtas_pci_write_config	,	F_14
iounmap	,	F_24
of_address_to_resource	,	F_18
KERN_INFO	,	V_38
last_busno	,	V_44
rtas_token	,	F_26
"read-pci-config"	,	L_6
__be32	,	T_4
printk	,	F_19
pci_controller	,	V_40
rtas_setup_phb	,	F_30
of_get_property	,	F_16
device_node	,	V_30
pci_dn	,	V_1
ulong	,	V_28
read_pci_config	,	V_23
rtas_read_config	,	F_2
"write-pci-config"	,	L_7
where	,	V_3
busno	,	V_19
write_pci_config	,	V_29
addr	,	V_10
PRG_CL_RESET_VALID	,	F_21
"bus-range"	,	L_10
"model"	,	L_1
EEH_PE_CFG_BLOCKED	,	V_17
EEH_IO_ERROR_VALUE	,	F_10
start	,	V_37
phb	,	V_21
"Python workaround: "	,	L_4
PCIBIOS_SUCCESSFUL	,	V_24
r	,	V_39
pe	,	V_15
KERN_ERR	,	V_36
__init	,	T_3
init_pci_config_tokens	,	F_25
pdn	,	V_5
